epsagon/events/pymongo.py

```diff
@@ -70,17 +76,38 @@ def update_response(self, response):
         :param response: Response from botocore
         :return: None
         """
-        for i in range(len(self.resource['metadata'].get('items', []))):
-            self.resource['metadata']['items'][i]['_id'] = str(
-                self.resource['metadata']['items'][i]['_id']
-            )
-
-        if self.resource['operation'] == 'insert_many':
-            self.resource['metadata']['inserted_ids'] = \
-                [str(x) for x in response.inserted_ids]
-        elif self.resource['operation'] == 'insert_one':
-            self.resource['metadata']['inserted_ids'] = \
-                [str(response.inserted_id)]
+
+        if self.resource['operation'] in PyMongoEvent.INSERT_OPERATIONS:
+            for i in range(len(self.resource['metadata'].get('items', []))):
```

Comment:
I would move each logic (of insert/query operations) to a separated method to handle the response data